code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -235,6 +235,7 @@
 	name = "hard-worn belt"
 	desc = "There's a lot you can tell about a man from his clothes - sometimes it's all he can afford, or maybe he chooses to wear something as a message... this belt, then, is a statement. Classy, but not too drab. Fashionable, but still useful. People look at this belt and think, 'My god. That belt is frighteningly well placed. The shades of beige that seem to flood over themselves, splayed across every notch... I fear many things, but I fear most the man who possesses such an incredible belt.'"
 	icon_state = "detective"
+	item_staet = "detective"
```

Comment:
collaborator code review and maintenance

Reply:
Fixed it for him right after I merged it.

Reply:
Why was this line even changed in the first place

Reply:
The world may never know